src/ar_tx_search.erl

```diff
@@ -34,71 +35,79 @@
 %% @doc Start a search node.
 start() ->
 	initDB(),
-	PID = spawn(fun server/0),
-	erlang:register(http_search_node, PID),
-	{ok, PID}.
+	ok.
```

Comment:
The module documentation (in the beginning of the file) needs an update.

Specifically, these parts should be removed:

```
%%% Access to the DB is wrapped in a server, which limits the number of
%%% concurrent requests to 1.
```

and

```
%%% Warning! Writes returns early but are serialized together with reads. If a
%%% write fails it will be logged, but the queue of reads and writes will
%%% continue.
```